doc/example-requests/adapter.http

```diff
@@ -9,6 +9,7 @@ GET {{baseURL}}/formats  HTTP/1.1
 Content-Type: application/json
 
 ### Perform Data Import JSON
+## DEPRECATED
```

Comment:
Why? Don't we still have that endpoint for preview purposes?

Reply:
Right, I forgot about that :D